lib/menu.rb

```diff
@@ -0,0 +1,16 @@
+require_relative './dish.rb'
+
+class Menu
+
+  DEFAULT_MENU = {biriyani: 5.00, uk_curry: 6.00, cheeseburger: 4.00, pizza: 5.00, side_of_salmonella: 1.95}
```

Comment:
Line is too long. [108/80]<br>Space inside { missing.<br>Space inside } missing.